support/routes/v2/isobot/index.ts

```diff
@@ -16,11 +23,37 @@ const token = config.get("slackbot.token")
 const botReceiver = new ExpressReceiver({ signingSecret, endpoints: "/" })
 export const isobotRouter = botReceiver.router
 
+// TODO: add slack ids of isomer user
+const ISOMER_USERS_ID = ["U01HTSFC0RY"]
```

Comment:
should we be hardcoding this? Or fetch it dynamically? Else every member movement in team will require a change

An alternative I can think of is to fetch members of the `isomer-team` and then check if the user is within this team. I believe the team will be more permanent than a user id

Reply:
ok this sounds good to me, i'll make this change!